### PR TITLE
Update the version number we identify as to 0.2.

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

File: contrib/rpm/buildah.spec [hunk 1]
```diff
@@ -87,5 +87,32 @@ make DESTDIR=%{buildroot} PREFIX=%{_prefix} install install.completions
...
+-   Add flag to remove containers on commit
+-   Improve buildah push man page and help information
+-   add a way to disable PTY allocation
+-   clarify --runtime-flag of run command
+-   Need \n for Printing untagged message
+-   Update to match newer storage and image-spec APIs
+-   Build imgtype independently
+-   Bump containers/storage and containers/image
+-   imgtype: do not log at Fatal level
+-   imgtype: add debugging, reexec, an optimization
```
Comment (nalind) [inline contrib/rpm/buildah.spec]: Let's prune out the changes that aren't user-visible.

Comment (rhatdan) [inline contrib/rpm/buildah.spec]: Fixed

**Comment by nalind** [comment]
LGTM pending CI.

**Comment by nalind** [comment]
@rh-atomic-bot r+ 0892c451927b118cb34c536e126f8552eb7a4d9f

**Pull request closed without merging**